src/ForeignController.sol

```diff
@@ -59,11 +59,11 @@ contract ForeignController is AccessControl {
     event MintRecipientSet(uint32 indexed destinationDomain, bytes32 mintRecipient);
     event RelayerRemoved(address indexed relayer);
 
+    event UniswapV3RouterSet(address indexed router);
     event UniswapV3PositionManagerSet(address indexed positionManager);
     event UniswapV3PoolLowerTickUpdated(address indexed pool, int24 lowerTick);
     event UniswapV3PoolUpperTickUpdated(address indexed pool, int24 upperTick);
-    event UniswapV3PoolTwapSecondsAgoUpdated(address indexed pool, uint32 swapTwapSecondsAgo);
-
+    event UniswapV3SwapMaxTickDeltaSet(address indexed pool, uint24 maxTickDelta);
```

Comment:
I think this event should also conform to the others and the function name even if maxTickDelta is only currently used for swaps:
```suggestion
    event UniswapV3PoolMaxTickDeltaSet(address indexed pool, uint24 maxTickDelta);
```